core/test/BatteryPluginHistoryTest.java

```diff
@@ -0,0 +1,101 @@
+import com.sonyericsson.chkbugreport.BugReportModule;
+import com.sonyericsson.chkbugreport.Context;
+import com.sonyericsson.chkbugreport.Section;
+import com.sonyericsson.chkbugreport.chart.ChartPluginRepo;
+import com.sonyericsson.chkbugreport.doc.Chapter;
+import com.sonyericsson.chkbugreport.plugins.battery.BatteryInfoPlugin;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotEquals;
+import static org.mockito.Mockito.*;
```

Comment:
Please use specific imports rather than wildcard

Reply:
Changed, although I would note the wildcard style is how all the mockito examples are (although likely out of laziness on their part)